app/views/organization_timeline_entries/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if can?(:read, @organization_timeline_entry) %>
+  <%- model_class = OrganizationTimelineEntry -%>
+  <div class="page-header">
+    <h1><%=t '.title', :default => model_class.model_name.human %></h1>
+  </div>
+
+  <dl class="dl-horizontal">
+    <dt><strong><%= model_class.human_attribute_name(:organization) %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.organization.name %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:entry_type).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.entry_type.capitalize %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:started_at).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.started_at %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:ended_at).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.ended_at %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:description).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.description %></dd>
+  </dl>
+  
+  <div class="form-actions">
+    <%= link_to t('.back', :default => t("helpers.links.back")),
+              organization_timeline_entries_path, :class => 'btn btn-default' %>
```

Comment:
This path doesn't exist.  You should change it to ":back" so it redirects to the previous page the user was on.